docs/system-admin-guide/file-storages/file-storage-troubleshooting/README.md

```diff
@@ -0,0 +1,51 @@
+  | Not allowed  | Outbound request method not allowed     | OpenProject sent wrong requests to the storage.  <br />This error can occur both in Nextcloud and OneDrive/Sharepoint integration. | Report this to [OpenProject community](https://community.openproject.org/projects/openproject/forums) or [support team](https://www.openproject.org/contact/). |
+  | Not found    | Outbound request destination not found  | OpenProject can not reach file storage platform:  <br />- This could be due to Storage provider being down:<br />- DNS problems <br />- Network problems (flaky) <br />- Local networks (Nexctloud specific setting that needs to enabled)  <br />This error can occur both in Nextcloud and OneDrive/Sharepoint integration. | See if you can access the file storage platform from your browser.  <br />For Nextcloud, see if Nextcloud settings are active if in local network. |
+  | Unauthorized | Outbound request not authorized         | - Authentification is failing<br /> - Application password was changed and not updated in OpenProject (Nextcloud OAuth settings are wrong and SP client secret or id is wrong).<br />- User has no access, can not login, no token can be negotiated.<br />  Server to server: the client secret might be wrong <br /> OpenProject User credentials might be wrong | Check the storages setup.<br />Check if the client secret (SP) or the OAuth setup is correct (in Nextcloud).<br />Check if the application password is correct. |
+  | Conflict     | *error_text_from_response*              | A folder or a file was created, which already exists on the storage platform, e.g. creating a folder with the same name. <br /> Can happen if for example a user manually created something on the storage platform. | Check in the storage platform if the folder already exists.  |
+  | Error        | Outbound request failed                 | An unexpected 500 error, e.g. TOS app was activated and OpenProject can not access storage anymore. <br /> Password configuration plugin may have caused problems. | See if file storage is working correctly. If it does, collect as much information as possible and contact [OpenProject community](https://community.openproject.org/projects/openproject/forums) or [support team](https://www.openproject.org/contact/). |
+
+
+
+
+If the suggested troubleshooting solutions did not resolve your issue, please reach out to the [OpenProject community](https://community.openproject.org/projects/openproject/forums) or [support team](https://www.openproject.org/contact/) for further assistance.
+
+## Unhealthy file storages e-mail notifications
+
+Once a file storage has been set-up 
```

Comment:
Is this section accidentally in the PR or a work in progress?

Reply:
Thanks for catching this. The section moved up in the page, but was originally written under Error Explanations and remained there by accident. I will remove it.